components/ui/Gnb.vue

```diff
@@ -12,15 +12,15 @@
                 background-color="#212121"
                 text-color="#fff"
                 active-text-color="#ffd04b">
-                <el-menu-item index="guide" >덱 가이드</el-menu-item>
+                <el-menu-item index="guide">덱 가이드</el-menu-item>
 
-                <el-submenu
+                <el-submenu v-if="$store.getters['seasons']"
```

Comment:
Codacy has a fix for the issue: [Insert `⏎···················`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
                <el-submenu
```